### PR TITLE
Bugfix/verbrauch

### PR DESCRIPTION
**Pull request by reox**
go back to `verbrauch` for 24h hourly data.
Clean up the API calls a bit.

### CODE REVIEW

File: custom_components/wnsm/statistics_sensor.py [hunk 1]
```diff
@@ -205,35 +205,43 @@ async def _import_statistics(self, smartmeter: Smartmeter, start: datetime, tota
...
+                _LOGGER.warning(f"Data starting at {start} does not contain granular data! Opt-in was not set back then. Skipping.")
                 continue
 
             # Can actually check, if the whole batch can be skipped.
             if consumption.get('consumptionMinimum') == 0 and consumption.get('consumptionMaximum') == 0:
-                _LOGGER.debug("Batch of data does not contain any consumption, skipping")
+                _LOGGER.debug(f"Batch of data starting at {start} does not contain any consumption, skipping")
                 continue
 
+            if 'values' not in consumption:
```
Comment (reox) [inline custom_components/wnsm/statistics_sensor.py:226]: Maybe this change here is not required: The `verbrauch` endpoint should always return values, but if no stuff was recorded or is not available, these values are zero. Thus it looks like that in contrary to `verbrauchRaw`, no `values` is indeed indicating an error.
What to do?

**Comment by reox** [comment]
I remove the `dateTo` from the function, because it is not required (anymore?). But where is that still defined in the test?

**Review by tixoman — COMMENTED** [review]
passt und funktioniert

**Review by tixoman — APPROVED** [review]

**Review by DarwinsBuddy — APPROVED** [review]
lgtm

**Pull request merged**